src/qml/pages/ChatPage.qml

```diff
@@ -75,6 +75,64 @@ Page {
                     color: Theme.secondaryTextColor
                 }
             }
+
+            ImageButton {
+                image: "More"
+                onClicked: {
+                    if (appState.isGroupChat) {
+                        if (appState.isAdmin) {
+                            groupChatAdminContextMenu.currentIndex = -1
+                            groupChatAdminContextMenu.open()
+                        } else {
+                            groupChatNotAdminContextMenu.currentIndex = -1
+                            groupChatNotAdminContextMenu.open()
+                        }
+                    } else {
+                        stdChatContextMenu.currentIndex = -1
+                        stdChatContextMenu.open()
+                    }
+                }
+
+                ContextMenu {
+                    id: stdChatContextMenu
+                    dropdown: true
+
+                    Action {
+                        text: qsTr("Delete chat")
+                        onTriggered: controllers.chats.addParticipant(/*userId*/)
```

Comment:
This shouldn't be commented, method has a parameter
Again, don't leave redundant comments

Reply:
+